syslog-ng-ctl/syslog-ng-ctl.c

```diff
@@ -411,17 +398,35 @@ slng_passwd(int argc, char *argv[], const gchar *mode)
   gint buff_size = 255;
   gchar buff[buff_size+1];
 
-  if (g_strcmp0(argv[1],"store")==0 && (argv[3]==NULL || strlen(argv[3])==0))
```

Comment:
argc should be checked for argument existence

Reply:
OK, added argc check